packages/tools/examples/videoTools/index.ts

```diff
@@ -156,18 +159,35 @@ addDropdownToToolbar({
   },
 });
 
+const useLocal = true;
```

Comment:
please remove useLocal

Reply:
Needed for testing purposes, and it is much cleaner to change a variable.  Ideally this should come from the environment variables so one can build either way.  However, it does need to default to false.

Reply:
You are hardcoding the port 5000 which is another problem